admin/static/js/metrics/metrics.es6.js

```diff
@@ -941,12 +941,32 @@ var AddonMetrics = function() {
 };
 
 
+// <+><+><+><><>+
+//   preprints   |
+// ><+><+><+<+><+
+
+var PreprintMetrics = function() {
+    var preprint_created = new keenAnalysis.Query("sum", {
+        eventCollection: "preprint_summary",
+        targetProperty: "provider.total",
+        groupBy: ["provider.name"],
+        interval: "daily",
+        timeframe: "this_365_days",
```

Comment:
365 days is probably going to get expensive. It should happen too frequently, but unless Product really is hoping for a year long graph, we probably don't need it. It's worth checking to see if a week or a month view would be acceptable.